src/main/reactjs/src/components/clerkTranslator/overview/ClerkTranslatorDetails.tsx

```diff
@@ -129,7 +146,7 @@ export const ClerkTranslatorDetails = () => {
       editDisabled={isViewMode}
       topControlButtons={
         <ControlButtons
-          onCancelBtnClick={openCancelDialog}
+          onCancelBtnClick={handleCancelBtnClick}
```

Comment:
mahollisesti `onCancel={handleCancel}` etc?

Reply:
Nuo kaikki muut handlerit ovat kyllä muotoa `handleXBtnClick` -> pitäisin tällaisena.

Reply:
Joo, 'etc' jäi vähän liian vagueksi, koitin sillä ilmaista että kaikille muille sama käsittely (onCancel, onEdit..). Mietin, että tarviiko parentin tietää tuosta ControlButtons implementaatiosta sen enempää kuin, että se tarvii esim. onCancel, joka sitten toki ControlButtons sisällä sidotaan napin klikkaukseen. Pieni nyanssihan tää

Reply:
Aivan. Joo, tää kuulostaa ihan hyvältä näin auki kirjoitettuna.